peachjam/models/citations.py

```diff
@@ -47,3 +45,36 @@ def from_extracted_citation(cls, citation):
                 },
             ],
         )
+
+
+class ExtractedCitation(models.Model):
+    citing_work = models.ForeignKey(
+        "peachjam.Work",
+        null=False,
+        on_delete=models.CASCADE,
+        related_name="citing_work",
+        verbose_name=_("citing work"),
+    )
+    target_work = models.ForeignKey(
+        "peachjam.Work",
+        null=False,
+        on_delete=models.CASCADE,
+        related_name="target_work",
+        verbose_name=_("target work"),
+    )
+
+    @classmethod
+    def for_citing_works(cls, work):
+        return (
+            cls.objects.prefetch_related("citing_work", "target_work")
+            .filter(citing_work=work)
+            .order_by("citing_work__title")
```

Comment:
If you're filtering on citing work, then the title will always be the citing work's title. So this should be ordering by the target work, not the citing work.